### DISCUSSION TITLE
How to do a "star rating" component?

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#808] [Q&A] [answered]
Hi

I'm trying to have a "star rating" component on my html page: 

Does someone have any advice / best practice on how to achieve that ?

ideas:
-use meter ? 
-use different inputs for each star?

Is there a working example somewhere?
Regards

**Comment by GalinhaLX** [comment]
for those who are interested, a working collection of rating stars

[https://codepen.io/GATIGA/pen/mdBaGGr](https://codepen.io/GATIGA/pen/mdBaGGr)



( I guess there's an issue: [https://github.com/mavoweb/mavo/issues/811#issue-1103134699](https://github.com/mavoweb/mavo/issues/811#issue-1103134699) )

so I found a turnaround using expressions for the name

**Comment by LeaVerou** [comment, marked as answer]
This is a bit of a hack, but you could do it with a `<meter>`, taking advantage of Mavo's special handling for editing `<meter>` elements. Here is a demo with a `<meter>` styled to look like a star rating widget: https://codepen.io/leaverou/pen/ZEXVVvj
You could use any other emoji, not just stars. Or any SVG icon!

**Reply by GalinhaLX** [reply]
broken codepen link

here's the new one from @LeaVerou https://codepen.io/leaverou/pen/gOGZZvw

**Comment by GalinhaLX** [comment]
Thanks for the example, stars are even more nice :)
